Registry/RegistryLog.py

```diff
@@ -0,0 +1,150 @@
+#!/bin/python
+
+#    This file is part of python-registry.
+#
+#   Copyright 2011, 2012 Willi Ballenthin <william.ballenthin@mandiant.com>
```

Comment:
I should go through yearly and updated the dates in these headers. @msuhanov you can also add your name to files you create.